cmd/gateway-azure.go

```diff
@@ -768,7 +764,7 @@ func (a *azureObjects) CompleteMultipartUpload(bucket, object, uploadID string,
 			return objInfo, traceError(PartTooSmall{
 				PartNumber: uploadedParts[i].PartNumber,
 				PartSize:   size,
-				PartETag:   uploadedParts[i].ETag,
+				PartETag:   azureToS3ETag(uploadedParts[i].ETag),
```

Comment:
The above two changes are not required.

Reply:
Why? Without it, it doesn't work

Reply:
Hum, maybe not finally, let me check again